lang/cs/Org.Apache.REEF.Common/Services/ServiceConfiguration.cs

```diff
@@ -66,22 +49,13 @@ public static ConfigurationModule ConfigurationModule
             }
         }
 
-        public IConfiguration TangConfig { get; private set; }
-    }
-
-    public class InjectedServices
-    {
-        [Inject]
-        public InjectedServices([Parameter(typeof(ServicesSet))] ISet<IService> services)
+        private ServiceConfiguration()
         {
-            Services = services;
         }
-
-        public ISet<IService> Services { get; set; }
     }
 
     [NamedParameter("Set of services", "servicesSet")]
-    class ServicesSet : Name<ISet<IService>>
+    class ServicesSet : Name<ISet<object>>
```

Comment:
I know, as we discussed, technically we can inject any object from driver to evaluator. Then shall we still call them as services? I have thrown this question in the dev mailing list following the previous discussions.